src/utils/ranges.js

```diff
@@ -0,0 +1 @@
+export default [['2712'],['2714'],['2716'],['271D'],['2721'],['2728'],['2733'],['2744'],['2747'],['274C'],['274E'],['2753','2754'],['2757'],['2763'],['2795','2796'],['27A1'],['27B0'],['27BF'],['2934'],['2B05','2B06'],['2B1B'],['2B50'],['2B55'],['3030'],['303D'],['1F004'],['1F0CF'],['1F170'],['1F17E','1F17F'],['1F18E'],['1F191','1F199'],['1F1E6','1F1FE'],['1F201'],['1F21A'],['1F22F'],['1F232','1F239'],['1F250'],['1F300','1F30C'],['1F30F'],['1F311'],['1F313','1F315'],['1F319'],['1F31F'],['1F330','1F331'],['1F334'],['1F337','1F34A'],['1F34C','1F34F'],['1F351','1F37B'],['1F380','1F392'],['1F3A0','1F3C3'],['1F3C6'],['1F3C8'],['1F3CA'],['1F3E0','1F3E3'],['1F3E5','1F3EF'],['1F419','1F429'],['1F42B','1F43D'],['1F440'],['1F442','1F464'],['1F466','1F46B'],['1F46E','1F4AC'],['1F4AE','1F4B5'],['1F4B8','1F4EB'],['1F4EE'],['1F4F0','1F4F4'],['1F4F6'],['1F4F9','1F4FB'],['1F503'],['1F50A','1F514'],['1F516','1F52B'],['1F52F','1F53C'],['1F550','1F55B'],['1F5FB','1F5FE'],['1F601','1F60F'],['1F612','1F613'],['1F616'],['1F618'],['1F61A'],['1F61C','1F61D'],['1F620','1F624'],['1F628','1F62A'],['1F62D'],['1F630','1F632'],['1F635','1F63F'],['1F645','1F64E'],['1F680'],['1F683','1F685'],['1F687'],['1F689'],['1F68C'],['1F693'],['1F69A'],['1F6A5'],['1F6A7','1F6AD'],['1F6B2'],['1F6B6'],['1F6BB','1F6BE'],['3297'],['3299'],['00A9'],['00AE'],['0023'],['002A'],['0030','0038'],['203C'],['2049'],['2122'],['2139'],['2194','2198'],['21A9'],['231A'],['2328'],['23CF'],['23E9','23EC'],['23F0'],['23F3'],['24C2'],['25AA'],['25B6'],['25C0'],['25FB','25FD'],['2600','2603'],['260E'],['2611'],['2614'],['2618'],['261D'],['2620'],['2622'],['2626'],['262A'],['262E'],['2638','2639'],['2648','2652'],['2660'],['2663'],['2665'],['2668'],['267B'],['267F'],['2692','2693'],['2696'],['2699'],['269B'],['26A0','26A1'],['26AA'],['26BD'],['26C4'],['26CE'],['26D4'],['26EA'],['26F2','26F3'],['26F5'],['26FA'],['26FD'],['2702'],['2705'],['2708'],['270A'],['270C'],['270F']];
```

Comment:
You could write these as hexadecimal literals, `0x2712` instead of strings, then avoid the number parsing coding in  index.js